adafruit_rsa/common.py

```diff
@@ -35,7 +25,7 @@ class NotRelativePrimeError(ValueError):
     """Raises if provided a and b not relatively prime."""
 
     def __init__(self, a, b, d, msg=None):
-        super(NotRelativePrimeError, self).__init__(
```

Comment:
I would need to look into this argument removal further (or get confirmed by someone who knows more), this came up in another one of these PRs too. I'll try to check into it more this weekend and see if I can verify that this is okay and intended.

Reply:
I think @kattni told me doing that shouldn't affect the functionality